core/src/character.rs

```diff
@@ -19,4 +20,5 @@ pub enum Character<'gc> {
     Text(Text<'gc>),
     Sound(#[collect(require_static)] SoundHandle),
     Video(Video<'gc>),
+    BinaryData(BinaryData<'gc>),
```

Comment:
Is there any particular reason why we're using `BinaryData<'gc>` instead of a `SwfSlice`? As far as I can tell the former just holds one of the latter.